pytest_django/fixtures.py

```diff
@@ -124,6 +125,7 @@ def _django_db_fixture_helper(transactional, request, django_db_blocker):
         from django.test import TestCase as django_case
 
     test_case = django_case(methodName='__init__')
+    test_case.multi_db = multi_db
     test_case._pre_setup()
     request.addfinalizer(test_case._post_teardown)
```

Comment:
I think a more universal approach would be to get access to the Django TestCase, where you then could set `multi_db` on.  I will create a PR for that.
/cc @pelme